hazelcast/src/test/java/com/hazelcast/topic/impl/reliable/ReliableTopicStressTest.java

```diff
@@ -0,0 +1,115 @@
+package com.hazelcast.topic.impl.reliable;
+
+
+import com.hazelcast.config.Config;
+import com.hazelcast.config.TopicConfig;
+import com.hazelcast.core.HazelcastInstance;
+import com.hazelcast.core.ITopic;
+import com.hazelcast.core.Message;
+import com.hazelcast.core.MessageListener;
+import com.hazelcast.test.AssertTask;
+import com.hazelcast.test.HazelcastSerialClassRunner;
+import com.hazelcast.test.HazelcastTestSupport;
+import com.hazelcast.test.TestThread;
+import com.hazelcast.test.annotation.NightlyTest;
+import org.junit.Before;
+import org.junit.Ignore;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.junit.runner.RunWith;
+
+import static java.util.concurrent.TimeUnit.SECONDS;
+import static org.junit.Assert.assertEquals;
+
+@RunWith(HazelcastSerialClassRunner.class)
+@Category(NightlyTest.class)
+@Ignore
```

Comment:
My bad.. because I was measuring test coverage and I don't want to run the stress test for that.